src/dscim/preprocessing/input_damages.py

```diff
@@ -738,12 +778,21 @@ def prep(
         ).expand_dims({"gcm": [gcm]})
 
         damages = damages.chunk(
-            {"batch": 15, "ssp": 1, "model": 1, "rcp": 1, "gcm": 1, "year": 10}
+            {
+                "batch": 15,
```

Comment:
why are we chunking by the same chunks in both concat function and this function?

Reply:
I have a different answer to this comment than the one I gave during the meeting. Only energy and labor post-processed damages need to run through the concat function because they are separated by batches. Chunks for energy and labor damages are set during concatenation. The function here is for processing mortality damages. It will output a single file concatenated across batches directly, so chunks need to be set when processing damages for mortality.

Reply:
that makes sense, thanks for clearing it up